### ISSUE TITLE
ModuleNotFoundError: No module named 'pubsub'

### ISSUE THREAD

**Issue by gitting-around** [INTO-CPS-Association/unifmu#11]
After installing unifmu and running in the shell:

`$ unifmu --help`

 I get the following error

```
Traceback (most recent call last):
  File "/usr/local/bin/unifmu", line 5, in <module>
    from unifmu.cli import main
  File "/usr/local/lib/python3.8/site-packages/unifmu/cli.py", line 3, in <module>
    from unifmu.gui import show_gui
  File "/usr/local/lib/python3.8/site-packages/unifmu/gui.py", line 5, in <module>
    from pubsub.pub import validate
ModuleNotFoundError: No module named 'pubsub'
```

(Running it on macos Catalina)

**Comment by gitting-around** [comment]
After the suggestion from Christian I installed [pubsub](https://pypi.org/project/pubsub/):

`$ pip install pubsub`

I also needed to do:

`$ pip install Pypubsub`
`$ pip install wxpython`

on my system for it to work.

**Comment by clegaard** [comment]
solved by f224340

To install gui dependencies use:
```
pip install .[gui]
```

A error message has been added to describe this if the gui is launched without these dependencies
